#local_planner: "TebLocalPlannerROS"
# local_planner: "PebbleLocalPlanner"
# local_planner: "RegPurePursuitPlanner"
parameter_name_linear: "max_vel_x"
parameter_name_angular: "max_vel_theta"

velocities:
  cruising: 0.5
  cleaning: 0.3
